mobile/src/main/java/org/openhab/habdroid/util/Util.java

```diff
@@ -281,6 +284,26 @@ public static void applyAuthentication(WebView webView, Connection connection, S
             webView.setHttpAuthUsernamePassword(Util.getHostFromUrl(url), "",
                     connection.getUsername(), connection.getPassword());
         }
+
+        webView.getSettings().setDomStorageEnabled(true);
+        webView.getSettings().setJavaScriptEnabled(true);
+        webView.getSettings().setSupportMultipleWindows(true);
+
+        webView.setWebChromeClient(new WebChromeClient() {
+            @Override
+            public boolean onCreateWindow(WebView view, boolean dialog, boolean userGesture,
+                    Message resultMsg) {
+                Message href = view.getHandler().obtainMessage();
+                view.requestFocusNodeHref(href);
+                String url = href.getData().getString("url");
+                if (url != null) {
+                    Context context = view.getContext();
+                    Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse(url));
+                    context.startActivity(browserIntent);
```

Comment:
Should catch having no browser installed: https://github.com/openhab/openhab-android/blob/master/mobile/src/main/java/org/openhab/habdroid/ui/AboutActivity.java#L334-L339